docs/Documentation/Development/Containers/index.md

```diff
@@ -53,7 +50,7 @@ FROM ubuntu:22.04
 RUN apt-get update -y && apt-get install python3 -y
 ```
 
-Images are normally built (or "bootstrapped") from a base image indicated by `FROM`. This base image is composed of one or more layers that will be pulled from the appropriate [container registry](registries.md) during buildtime. In this example, version 22.04 of the Ubuntu operating system is specified as the base image. Docker pulls from Ubuntu's [DockerHub container registry](https://hub.docker.com/_/ubuntu) by default. The ability to use a different base image provides a way to use packages which may work more easily on a specific operating system distribution. For example, the Linux distribution on Eagle is CentOS, so building the above image would allow the user to install packages from Ubuntu repositories.
+Images are normally built (or "bootstrapped") from a base image indicated by `FROM`. This base image is composed of one or more layers that will be pulled from the appropriate [container registry](registries.md) during buildtime. In this example, version 22.04 of the Ubuntu operating system is specified as the base image. Docker pulls from Ubuntu's [DockerHub container registry](https://hub.docker.com/_/ubuntu) by default. The ability to use a different base image provides a way to use packages which may work more easily on a specific operating system distribution. For example, the Linux distribution on Kestrel is CentOS, so building the above image would allow the user to install packages from Ubuntu repositories.
```

Comment:
```suggestion
Images are normally built (or "bootstrapped") from a base image indicated by `FROM`. This base image is composed of one or more layers that will be pulled from the appropriate [container registry](registries.md) during buildtime. In this example, version 22.04 of the Ubuntu operating system is specified as the base image. Docker pulls from Ubuntu's [DockerHub container registry](https://hub.docker.com/_/ubuntu) by default. The ability to use a different base image provides a way to use packages which may work more easily on a specific operating system distribution. For example, the Linux distribution on Kestrel is RedHat, so building the above image would allow the user to install packages from Ubuntu repositories.
```